src/ResourceManager/Resources/Commands.Resources/help/New-AzureRmRoleDefinition.md

```diff
@@ -56,29 +56,37 @@ The input role definition MAY contain the following properties:
 
 1) NotActions: the set of operations that must be excluded from the Actions to determine the effective actions for the custom role.
 If there is a specific operation that you do not wish to grant access to in a custom role, it is convenient to use NotActions to exclude it, rather than specifying all operations other than that specific operation in Actions.
+2) DataActions: the set of data operations to which the custom role grants access.
+3) NotDataActions: the set of operations that must be excluded from the DataActions to determine the effective dataactions for the custom role.
+If there is a specific data operation that you do not wish to grant access to in a custom role, it is convenient to use NotDataActions to exclude it, rather than specifying all operations other than that specific operation in Actions.
 
 NOTE: If a user is assigned a role that specifies an operation in NotActions and also assigned another role grants access to the same operation - the user will be able to perform that operation.
 NotActions is not a deny rule - it is simply a convenient way to create a set of allowed operations when specific operations need to be excluded.
 
 Following is a sample json role definition that can be provided as input
-        {
-        "Name": "Contoso On-call",
-        "Description": "Can monitor compute, network and storage, and restart virtual machines",
-        "Actions": \[
-        "Microsoft.Compute/*/read",
-        "Microsoft.Compute/virtualMachines/start/action",
-        "Microsoft.Compute/virtualMachines/restart/action",
-        "Microsoft.Compute/virtualMachines/downloadRemoteDesktopConnectionFile/action",
-        "Microsoft.Network/*/read",
-        "Microsoft.Storage/*/read",
-        "Microsoft.Authorization/*/read",
-        "Microsoft.Resources/subscriptions/resourceGroups/read",
-        "Microsoft.Resources/subscriptions/resourceGroups/resources/read",
-        "Microsoft.Insights/alertRules/*",
-        "Microsoft.Support/*"
+{
+        "Name": "Updated Role",
+        "Description": "Can monitor all resources and start and restart virtual machines",
+        "Actions":
+        \[
+            "*/read",
+            "Microsoft.ClassicCompute/virtualmachines/restart/action",
+            "Microsoft.ClassicCompute/virtualmachines/start/action"
         \],
-        "AssignableScopes": \["/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx","/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy"\]
-        }
+        "NotActions":
+        \[
+            "*/write"
+        \],
+        "DataActions":
+        \[
+            "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read"
+        \],
+        "NotDataActions":
+        \[
+            "Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write"
+        \],
+        "AssignableScopes": \["/subscriptions/4004a9fd-d58e-48dc-aeb2-4a4aec58606f"\]
```

Comment:
Change this to xxxx format - this looks like a real subscription.

Reply:
updating it